fhir-examples/src/main/java/com/ibm/fhir/examples/Index.java

```diff
@@ -0,0 +1,62 @@
+package com.ibm.fhir.examples;
+
+public enum Index {
+    /**
+     * Both R4 spec and IBM generated examples
+     */
+    ALL_JSON("/all-json.txt"),
```

Comment:
I'm looking at the format. 
perhaps unifying the layout. 

[PROVIDER]-[TYPE]-[FORMAT].txt
provider =  IBM | SPEC | third-party ? 
type = all | minimal | performance ? 
format = xml | json 

just a thought...

Reply:
we used to have "TestType" and "Format" as different fields, but I think it just got in the way.  I think we'd like to be able to create a single index file with a mix of all 3 of those in it.
For now, I left the index files as-is and just named them to indicate what was in them, but now there is nothing stopping us from defining one with a mix of JSON and XML.  (and we already have ones with a mix of IBM and SPEC).
If we weren't files-system bound, we could just keep all the files in a flat structure and tag them as appropriate, but that definitely makes it hard on eclipse (and programs like it) when you open that directory...

Reply:
no problem. just food for thought.